frontend/public/components/monitoring.tsx

```diff
@@ -169,14 +169,32 @@ const Label = ({k, v}) => <div className="co-m-label co-m-label--expand" key={k}
   <span className="co-m-label__value">{v}</span>
 </div>;
 
-const Graph = ({metric = undefined, numSamples, rule}) => {
+const graphStateToProps = ({UI}) => ({hideGraphs: !!UI.getIn(['monitoring', 'hideGraphs'])});
+
+const ToggleGraph_ = ({hideGraphs, toggle}) => {
+  const onClick = e => {
+    toggle();
+    e.target.blur();
```

Comment:
I'm unclear why we want to blur

Reply:
Actually, I think it's probably better to not blur, so I've removed this now. It was to remove the `:focus` styling, but actually we probably want to leave it to be consistent with other `btn-link`s. Thanks!